internal/rbd/nodeserver.go

```diff
@@ -173,12 +170,7 @@ func populateRbdVol(
 
 		disableInUseChecks = true
 	}
-
-	rv, err := genVolFromVolumeOptions(ctx, req.GetVolumeContext(), disableInUseChecks, true)
-	if err != nil {
-		return nil, status.Error(codes.Internal, err.Error())
-	}
-
+	rv := &rbdVolume{}
```

Comment:
**Nit:** This works, but creating an empty object before overwriting it is a little ugly.

I prefer to use this instead:
```go
var rv *rbdVolume
```